### PR TITLE
Add mapper 2.1 support

### PR DESCRIPTION
**Pull request by renchenglei**
This change is to help support mapper 2.1

Tracked-On: OAM-92929
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

### CODE REVIEW

**Review by feijiang1 — APPROVED** [review]

**Review by shyjumon-n — APPROVED** [review]
Looks good to me, graphics team to final review

File: groups/device-specific/caas/product.mk [hunk 1]
```diff
@@ -21,6 +21,7 @@ PRODUCT_PACKAGES += android.hardware.keymaster@3.0-impl \
                     android.hardware.camera.provider@2.4-impl \
                     android.hardware.camera.provider@2.4-service \
                     android.hardware.graphics.mapper@2.0-impl \
+                    android.hardware.graphics.mapper@2.0-impl-2.1 \
```
Comment (JeevakaPrabu) [inline groups/device-specific/caas/product.mk:24]: Why do we need android.hardware.graphics.mapper@2.0-impl when we have added  android.hardware.graphics.mapper@2.0-impl-2.1 package?

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

File: groups/device-specific/caas/product.mk (hunk 1 above)
Comment (renchenglei) [reply groups/device-specific/caas/product.mk:24]: I suppose this is a additional support for mapper. If there is some issues in mapper 2.1, we still could fall back to mapper 2.0.

> Why do we need [android.hardware.graphics.mapper@2.0-impl](mailto:android.hardware.graphics.mapper@2.0-impl) when we have added android.hardware.graphics.mapper@2.0-impl-2.1 package?

I suppose this is an additional support for mapper. If there are some issues in mapper 2.1, we still could fall back to mapper 2.0.

Comment (JeevakaPrabu) [reply groups/device-specific/caas/product.mk:24]: Could you please check the boot and other use cases with only android.hardware.graphics.mapper@2.0-impl-2.1 ?
I believe android.hardware.graphics.mapper@2.0-impl is not needed at all

Comment (renchenglei) [reply groups/device-specific/caas/product.mk:24]: @shyjumon-n, could you help confirm this? Especially the CTS on your side. Thanks a lot!

**Pull request closed without merging**